README.md

```diff
@@ -224,7 +253,7 @@ of CICE will be in the `ice` subdirectory.
 
 ## Running the Model
 
-If you have set up the modules system to use the `/g/data3/hh5/public/modules` folder, a simple `module load **payu**` should give you access to the **payu** system.
+If you have set up the modules system to use the `/g/data3/hh5/public/modules` folder, a simple `module load payu` should give you access to the **payu** system.
```

Comment:
`module load conda/analysis3-unstable`

Reply:
unstable? Will it still be in unstable by the time we release this document? But okay, I can change it for now.

Reply:
Yeah it will be unstable until next quarter. We never update/add packages to stable